docs/ovn_k8s.conf.5

```diff
@@ -72,6 +92,9 @@ The Kubernetes API authentication token (not required if kubeconfig is given).
 .SH [OvnSouth]
 .TP
 \fBaddress\fR=
+This is the url used to access the southbound database server. The scheme may be ssl or tcp.
+When ssl is used, the certs must be provided. The host must be a host IP address, not name.
+The port must be 6441. E.g., ssl://1.2.3.4:6641
```

Comment:
The port is usually 6641. Almost always. But one can configure northd to use a different one I believe. 'must' can be misleading.

Reply:
@rajatchopra I'll change it